tests/test_pipeline_images.py

```diff
@@ -76,35 +76,71 @@ def test_thumbnail_name(self):
                          'thumbs/50/850233df65a5b83361798f532f1fc549cd13cbe9.jpg')
 
     def test_convert_image(self):
+        # tests for old API
+        with warnings.catch_warnings(record=True) as w:
```

Comment:
could you please move "tests for old API" to a separate test method (or multiple test methods)?